gcp_flowlogs_reader/gcp_flowlogs_reader.py

```diff
@@ -20,25 +21,25 @@
 
 
 class FlowRecord:
-    __slots__ = [
-        'src_ip',
-        'src_port',
-        'dest_ip',
-        'dest_port',
-        'protocol',
-        'start_time',
-        'end_time',
-        'bytes_sent',
-        'packets_sent',
-        'rtt_msec',
-        'reporter',
-        'src_instance',
-        'dest_instance',
-        'src_vpc',
-        'dest_vpc',
-        'src_location',
-        'dest_location',
-    ]
+    src_ip: Union[IPv4Address, IPv6Address]
+    src_port: int
+    dest_ip: Union[IPv4Address, IPv6Address]
+    dest_port: int
+    protocol: int
+    start_time: datetime
+    end_time: datetime
+    bytes_sent: int
+    packets_sent: int
+    rtt_msec: Optional[int]
+    reporter: Any
```

Comment:
https://cloud.google.com/vpc/docs/flow-logs#record_format says this will be a `str`